tests/test_helpers/fabricators/user_fabricator.go

```diff
@@ -0,0 +1,32 @@
+package fabricators
+
+import (
+	"golang.org/x/crypto/bcrypt"
+
+	"go-crawler-challenge/models"
+
+	"github.com/onsi/ginkgo"
+)
+
+func FabricateUser(email string, plainPassword string) (user *models.User) {
```

Comment:
This function does a bit costly operation. It creates the user first(`models.AddUser(user)`), then search the same created user from the database again(`models.GetUserById(userId)`).  Is is possible to return the user instead of id for `AddUser(user)` function, then we don't to  search (`models. GetUserById (userId)`) it again?

Reply:
@junan If we remove `GetUserById` from here after getting `id` from `AddUser`, we still have searching again inside `AddUser` 🤔.
As the original function for inserting is returning just only `id` 😥
```
# github.com/beego/beego/v2@v2.0.1/client/orm/types.go
Insert(md interface{}) (int64, error)
```

Reply:
@Lahphim I guess you don't need to `GetUserById` function. I did simmilar, I get the user `record` without searching it aggain with id https://github.com/junan/google-scraper/blob/develop/tests/fabricators/user_fabricator.go#L11-L25